plugins/jira-dashboard-backend/config.d.ts

```diff
@@ -69,6 +74,17 @@ export interface Config {
            * Optional email suffix used for retrieving a specific Jira user in a company. For instance: @your-company.com. If not provided, the user entity profile email is used instead.
            */
           userEmailSuffix?: string;
+
+          /**
+           * Optional default filters for Jira queries
+           */
+          defaultFilters?: Filter[];
         }[];
       };
 }
+/** Represents a filter for Jira queries */
+export type Filter = {
```

Comment:
When I try to run the code, I get this error "Error: Invalid configuration schema in ../../plugins/jira-dashboard-backend/config.d.ts, additional symbol definitions are not allowed, found 'Filter'"

Reply:
You will have to "inline" the filter definition. 

https://backstage.io/docs/conf/defining/

Reply:
Done👍